src/views/edit/edit-tipp-view.vue

```diff
@@ -31,7 +31,7 @@
             v-if="id"
             :label="$t('component.general.uuid.label')"
             :value="uuid"
-            path="/title"
+            path="/#/title"
```

Comment:
This does not work for creating an URL which points to the REST API for example. We do not want a link to this client.

This should  be `/package-title` though. Curiously I think this had been fixed already on another branch...